can/interfaces/pcan/pcan.py

```diff
@@ -304,6 +312,21 @@ def bits(n):
 
         return complete_text
 
+    def get_api_version(self):
+        error, value = self.m_objPCANBasic.GetValue(PCAN_NONEBUS, PCAN_API_VERSION)
+        if error != PCAN_ERROR_OK:
+            raise CanInitializationError(f"Failed to read pcan basic api version")
+
+        return version.parse(value.decode("ascii"))
+
+    def check_api_version(self):
+        apv = self.get_api_version()
+        if apv < MIN_PCAN_API_VERSION:
+            raise CanInitializationError(
+                f"Minimum version of pcan api is {MIN_PCAN_API_VERSION}."
+                f" Installed version is {apv}. Consider upgrade of pcan basic package"
+            )
```

Comment:
We could also log a warning and try to continue anyway. Do you know more about what is imposing the current version requirement? Will every user always need that same minimum or is it only for some functionality?

Overall though, this seems like a good approach to me.

Reply:
Ok, a warning might be sufficient here. I will change this.
There is a document inside the PCAN Basic package: https://www.peak-system.com/fileadmin/media/files/pcan-basic.zip
Here it is described what version of the API is required for several functions to work. For the Parameter `PCAN_ALLOW_ ERROR_FRAMES` the version `4.2.0` or higher is required so i made this the minimum api version. I did not check if there were other parameters used with higer minimum version. It could be a good idea to attach the minimum version to every parameter so that it could be checked, but i think this is to much for this pr at the moment.